addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -23,6 +23,9 @@
         border: none !important;
```

Comment:
Commit message seems very extensive compared to the amount of code changes, are we really doing all of that? If yes, quite impressive what you can achieve with just carefully placed classes/css :smile: